app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -61,180 +64,240 @@
+
+    /** Emits a boolean indicating whether activities are being fetched from the API. */
+    Observable<Boolean> isFetchingActivities();
+
+    /** Emits a boolean that determines if a logged-out, empty state should be displayed. */
+    Observable<Boolean> loggedOutEmptyStateIsVisible();
+
+    /** Emits a logged-in user with zero activities in order to display an empty state. */
+    Observable<Boolean> loggedInEmptyStateIsVisible();
+
+    /** Emits a list of unanswered surveys to be shown in the user's activity feed */
+    Observable<List<SurveyResponse>> surveys();
   }
 
-  final class ViewModel extends ActivityViewModel<ActivityFeedActivity> implements ActivityFeedAdapter.Delegate,
-    Inputs, Outputs {
+  final class ViewModel extends ActivityViewModel<ActivityFeedActivity> implements
+    ActivityFeedViewModel.Inputs, ActivityFeedViewModel.Outputs {
 
     private final ApiClientType client;
+    private final CurrentConfigType currentConfig;
     private final CurrentUserType currentUser;
 
     public ViewModel(final @NonNull Environment environment) {
       super(environment);
 
-      client = environment.apiClient();
-      currentUser = environment.currentUser();
-
-      goToDiscovery = discoverProjectsClick;
-      goToLogin = loginClick;
-      goToProjectUpdate = projectUpdateClick;
-      goToProject = Observable.merge(
-        friendBackingClick,
-        projectStateChangedClick,
-        projectStateChangedPositiveClick,
-        projectUpdateProjectClick
+      this.client = environment.apiClient();
+      this.currentConfig = environment.currentConfig();
+      this.currentUser = environment.currentUser();
+
+      this.goToDiscovery = this.discoverProjectsClick;
+      this.goToLogin = this.loginClick;
+      this.goToProjectUpdate = this.projectUpdateClick;
+      this.goToSurvey = this.surveyClick;
+      this.goToProject = Observable.merge(
+        this.friendBackingClick,
+        this.projectStateChangedClick,
+        this.projectStateChangedPositiveClick,
+        this.projectUpdateProjectClick
       )
         .map(Activity::project);
 
+      final Observable<Boolean> surveyFeatureEnabled = this.currentConfig.observable()
+        .map(config -> coalesce(config.features().get(FeatureKey.ANDROID_SURVEYS), false));
+
+      final Observable<List<SurveyResponse>> responses = Observable.combineLatest(
+          resume,
+          this.currentUser.isLoggedIn(),
+          Pair::create
+        )
+        .filter(eventAndLoggedIn ->
+          eventAndLoggedIn.first == ActivityEvent.RESUME && eventAndLoggedIn.second
+        )
+        .compose(Transformers.combineLatestPair(surveyFeatureEnabled))
+        .switchMap(lifecycleAndLoggedInAndEnabled ->
+          lifecycleAndLoggedInAndEnabled.second
+            ? this.client.fetchUnansweredSurveys()
+            : Observable.just(Collections.emptyList())
+        );
+
+      responses
+        .subscribe(this.surveys::onNext);
```

Comment:
let's just attach this to the observable above and get rid of the local. only reason to break into a local is when you need to user it elsewhere.